common/views/components/SubNavigation/SubNavigation.tsx

```diff
@@ -46,53 +42,58 @@ const SubNavigation: FunctionComponent<Props> = ({
-                legacyBehavior
-              >
-                <NavItemInner
-                  variant={variant}
-                  selected={isSelected}
-                  aria-current={isSelected ? 'page' : 'false'}
-                  onClick={e => {
-                    if (!isSelected) {
-                      (e.target as HTMLDivElement).scrollIntoView({
-                        behavior: 'smooth',
-                        inline: 'center',
-                        block: 'nearest',
-                      });
-                    }
-                  }}
+    <>
+      <Wrapper aria-label={label}>
+        <TabsContainer data-test-id="sub-nav-tab-container">
+          {items.map(item => {
+            const isSelected = currentSection === item.id;
+            return (
+              <Tab data-test-id={item.id} key={item.id}>
+                <Link
+                  scroll={false}
+                  passHref
+                  href={typeof item.url === 'string' ? item.url : item.url.href}
+                  as={typeof item.url === 'string' ? undefined : item.url.as}
+                  legacyBehavior
                 >
-                  {item.icon && (
-                    <Space
-                      as="span"
-                      h={{ size: 's', properties: ['margin-right'] }}
-                    >
-                      <IconWrapper>
-                        <Icon icon={item.icon} />
-                      </IconWrapper>
-                    </Space>
-                  )}
-                  {item.name}
-                </NavItemInner>
-              </Link>
-            </Tab>
-          );
-        })}
-      </TabsContainer>
-
-      {hasDivider && <TabNavBottomBorder />}
-    </Wrapper>
+                  <NavItemInner
+                    variant={variant}
+                    selected={isSelected}
+                    aria-current={isSelected ? 'page' : 'false'}
+                    onClick={e => {
+                      if (!isSelected) {
+                        (e.target as HTMLDivElement).scrollIntoView({
+                          behavior: 'smooth',
+                          inline: 'center',
+                          block: 'nearest',
+                        });
+                      }
+                    }}
+                  >
+                    {item.icon && (
+                      <Space
+                        as="span"
+                        h={{ size: 's', properties: ['margin-right'] }}
+                      >
+                        <IconWrapper>
+                          <Icon icon={item.icon} />
+                        </IconWrapper>
+                      </Space>
+                    )}
+                    {item.name}
+                  </NavItemInner>
+                </Link>
+              </Tab>
+            );
+          })}
+        </TabsContainer>
+      </Wrapper>
+      {hasDivider && (
+        <DividerWrapper>
+          <Divider lineColor="neutral.300" />
```

Comment:
I think making it a background color was [a conscious change for a11y reasons](https://github.com/wellcomecollection/wellcomecollection.org/commit/3f3f08ecb500ba22d69a391dc9a6c8b8c8d76b65#diff-0d5e24a6e9a6786f25071160b2fdf270f158238c37c2e1b8d2ee617e809d1bf8) (@AMBacelar could maybe speak to it), we talked about it a bit on a FE catchup.

Although if we keep background colour it'd be good to use `theme.color('neutral.300')`.

Reply:
[Addressed above](https://github.com/wellcomecollection/wellcomecollection.org/pull/9408/files#r1135519223) after conversation with @AMBacelar and @rcantin-w